core/src/main/java/bio/terra/pearl/core/model/workflow/EventClass.java

```diff
@@ -2,14 +2,15 @@
 
 import bio.terra.pearl.core.service.consent.EnrolleeConsentEvent;
 import bio.terra.pearl.core.service.kit.KitStatusEvent;
+import bio.terra.pearl.core.service.survey.EnrolleeSurveyEvent;
 import bio.terra.pearl.core.service.workflow.BaseEvent;
 import bio.terra.pearl.core.service.workflow.EnrolleeCreationEvent;
 import bio.terra.pearl.core.service.workflow.PortalRegistrationEvent;
 
 public enum EventClass {
     ENROLLEE_CREATION_EVENT(EnrolleeCreationEvent.class),
     PORTAL_REGISTRATION_EVENT(PortalRegistrationEvent.class),
-    ENROLLEE_SURVEY_EVENT(EnrolleeCreationEvent.class),
+    ENROLLEE_SURVEY_EVENT(EnrolleeSurveyEvent.class),
```

Comment:
Unrelated to the fix in this PR, but I noticed this while tracing through the code